torch/quantization/_numeric_suite_fx.py

```diff
@@ -51,6 +46,33 @@ def load_arg(a):
     return model
 
 
+def _get_logger_dict_helper_fx(model, target_dict):
+    modules = dict(model.named_modules())
+
+    for node in model.graph.nodes:
+        if node.op == "call_module":
+            if isinstance(modules[node.target], Logger):
+                input_node = node.args[0]
+                if (
+                    input_node.op == "call_function"
+                    and input_node.target.__name__ == "quantize_per_tensor"
```

Comment:
you can do input_node.target == torch.quantize_per_tensor I think

Reply:
Yes, it works.Thanks!